framework/assets/yii.js

```diff
@@ -232,49 +230,41 @@ window.yii = (function ($) {
 
             var activeFormData = $form.data('yiiActiveForm');
             if (activeFormData) {
-                // remember who triggers the form submission. This is used by yii.activeForm.js
+                // Remember the element triggered the form submission. This is used by yii.activeForm.js.
                 activeFormData.submitObject = $e;
             }
 
-            // temporarily add hidden inputs according to data-params
-            if (params && $.isPlainObject(params)) {
-                $.each(params, function (idx, obj) {
```

Comment:
`idx` and `obj` are not acceptable here. Params are stored in `name` - `value` structure.